### PR TITLE
WinUI-2902 : Added the getting started content in the busyindicator control.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Prepared getting started content in the BusyIndicator control.

### CODE REVIEW

File: winui/BusyIndicator/Getting-Started.md [hunk 1]
```diff
@@ -1,12 +1,56 @@
...
+    xmlns:local="using:GettingStarted"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:notification="using:Syncfusion.UI.Xaml.Notifications"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Grid>
+      <notification:SfBusyIndicator IsActive="True"
+                   AnimationType="DottedCircularFluent"
+                   BusyContent="DottedCircularFluent">
```
Comment (Manivannan-E) [inline winui/BusyIndicator/Getting-Started.md:37]: @Priyanka-Vijayasankar Change the BusyContent value as Loading.

File: winui/BusyIndicator/Getting-Started.md [hunk 2]
```diff
@@ -1,12 +1,56 @@
...
+
+// Creating an instance of the BusyIndicator control.
+SfBusyIndicator busyIndicator = new SfBusyIndicator();
+
+// Activating the SfBusyIndicator.
+busyIndicator.IsActive = true;
+busyIndicator.AnimationType = BusyIndicatorAnimationType.DottedCircularFluent;
+busyIndicator.BusyContent = "DottedCircularFluent";
+
+{% endhighlight %}
```
Comment (Manivannan-E) [inline winui/BusyIndicator/Getting-Started.md:53]: @Priyanka-Vijayasankar Add two separate headings.

- Setting Animation Type
- Adding content

File: winui/BusyIndicator/Getting-Started.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Getting-Started.md]: Changed the BusyContent as loading.

File: winui/BusyIndicator/Getting-Started.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Getting-Started.md:115]: Added two separate headings as mentioned above.

File: winui/BusyIndicator/Getting-Started.md [hunk 3]
```diff
@@ -1,12 +1,118 @@
...
+// Activating the SfBusyIndicator.
+busyIndicator.IsActive = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Animation Type in BusyIndicator
+
+SfBusyIndicator provides 7 predefined animation types like DottedCircularFluent, DottedCircle, DottedLinear and so on. User can select any one of the animation types using `AnimationType` property.
+
```
Comment (Manivannan-E) [inline winui/BusyIndicator/Getting-Started.md:55]: @Priyanka-Vijayasankar The BusyIndicator control

Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Getting-Started.md:54]: Changed the content as mentioned above.

File: winui/BusyIndicator/Getting-Started.md [hunk 4]
```diff
@@ -1,12 +1,118 @@
...
+busyIndicator.IsActive = true;
+busyIndicator.AnimationType = BusyIndicatorAnimationType.DottedCircularFluent;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Adding Content in BusyIndicator
+
+SfBusyIndicator provides option to set the content that indicates the information related to loading. This can be done using `BusyContent` property.
+
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Getting-Started.md:75]: Changed the content as mentioned above.

**Comment by Priyanka-Vijayasankar** [comment]
rebuild

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/BusyIndicator/Getting-Started.md [hunk 5]
```diff
@@ -1,12 +1,114 @@
 ---
 layout: post
-title: Getting Started with WinUI BusyIndicator control | Syncfusion
-description: Learn all about getting started with the Syncfusion WinUI BusyIndicator(SfBusyIndicator) control, its elements, and more here.
+title: Getting started with WinUI BusyIndicator control | Syncfusion
+description: Learn here all about getting started with the Syncfusion WinUI BusyIndicator(SfBusyIndicator) control and its basic features.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/Getting-Started.md:4]: description: Learn all about getting started with the Syncfusion WinUI BusyIndicator(SfBusyIndicator) control and its basic features here.

File: winui/BusyIndicator/Getting-Started.md [hunk 6]
```diff
@@ -1,12 +1,114 @@
...
+
+// Activating the SfBusyIndicator.
+busyIndicator.IsActive = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Animation Type in BusyIndicator
+
+The BusyIndicator control provides 7 predefined animation types like `DottedCircularFluent`, `DottedCircle`, `DottedLinear` and so on. User can select any one of the animation types using `AnimationType` property.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/Getting-Started.md:53]: The BusyIndicator control provides 7 predefined animation types like `DottedCircularFluent`, `DottedCircle`, `DottedLinear`, and so on. Users can select any one of the animation types using the `AnimationType` property.

File: winui/BusyIndicator/Getting-Started.md [hunk 7]
```diff
@@ -1,12 +1,114 @@
...
+{% highlight C# %}
+
+SfBusyIndicator busyIndicator = new SfBusyIndicator();
+busyIndicator.IsActive = true;
+busyIndicator.BusyContent = "Loading";
+
+{% endhighlight %}
+{% endtabs %}
+
+The following code example gives you the complete code of above configurations.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/Getting-Started.md:93]: The following code example gives you the complete code of above configurations:

File: winui/BusyIndicator/Getting-Started.md (hunk 5 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Getting-Started.md:4]: Changed the content description as mentioned above.

File: winui/BusyIndicator/Getting-Started.md (hunk 6 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Getting-Started.md:53]: Changed the animation type content as mentioned above.

File: winui/BusyIndicator/Getting-Started.md (hunk 7 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Getting-Started.md:93]: Changed the content as mentioned above.

**Pull request merged**